packages/odyssey-react-mui/src/labs/SideNav/SortableList/SortableList.tsx

```diff
@@ -100,8 +100,9 @@ export const SortableList = <T extends BaseItem>({
         if (over && active.id !== over?.id) {
           const activeIndex = items.findIndex(({ id }) => id === active.id);
           const overIndex = items.findIndex(({ id }) => id === over.id);
-          if (items[overIndex].isSortable !== false)
+          if (items[overIndex].isSortable) {
             onChange(parentId, activeIndex, overIndex);
```

Comment:
@ganeshsomasundaram-okta Is `onChange` optional? I would assume it would be if `isSortable` is optional

Reply:
@bryancunningham-okta  onChange is an internal function needed to update the Sidenav items state when the list is reordered. We only get into this SortableList is the sideNavItem has `isSortable` set to `true`.  It doesn't makes sense to make the Parent list as Sortable but set all the child items to be not sortable.
There is also `onSort` which is a user passed optional callback which gets called when an item is reordered.

Reply:
Oh, so we defined `onChange`. It's not provided by the consumer?